content/module-reference/processing-modules/color-equalizer.md

```diff
@@ -1,7 +1,159 @@
+
+With saturation threshold practically disabled (at 0.0%), visualization indicator shows that the leaves are almost completely selected:
+
+![ce_saturation_treshold_1](https://github.com/user-attachments/assets/92e77378-d9e7-4a8a-9590-75ebba05a9cf)
+
+With a 20% saturation threshold, _only part of the green with higher saturation_ in the leaves _is selected_ (the red area):
+
+![ce_saturation_treshold_2](https://github.com/user-attachments/assets/ca620d6a-2cfc-4914-9cb9-cc1c4acc1e7b)
+
+By manipulating _only that high saturation area_, we have now enhanced the texture of the leaves:
+
+![ce_saturation_treshold_4](https://github.com/user-attachments/assets/3a2d7560-a108-42fd-a023-e4c813b635f2)
+
+: Set the upper bound for the guided filter's saturation threshold. Decrase the value to allow changes to areas with low chromaticity. Increase the value to restrict changes to areas of high chromaticity.
+
+contrast
+
+Contrast slider regulates the steepness of the curve defined by saturation threshold:
+
+contrast value -1:
+
+![ce_contrast_1](https://github.com/user-attachments/assets/b4883918-3132-4536-8dac-002690610b14)
+
+contrast value 1. The transition is much sharper. Thin green curve in the visualization indicator represents steepness of transition:
+
+![ce_contrast_2](https://github.com/user-attachments/assets/afb72f21-e7bc-4921-8de0-d6b916b14ba0)
+
+: Set the color contrast for the guided filter. Incrase th value to favor sharp transitions between colors and increase color contrast. Decrease the value for smoother color transition.
+
+effect radius
+
+Radius of the guided filter which is used for pixel manipulation to smooth the result while preserving the edges. 
+This can be useful for both preserving detail or smoothing the result, depending on the subject and the manipulation  that have been performed.
+
+In this case, we want to lighten the pumpkin. Without the guided filter or with a very low radius, we are loosing the details:
+
+![ce_effect_radius_4](https://github.com/user-attachments/assets/e16f7d9d-e960-40a2-bb49-f884070d8b52)
+
+If we then increase the radius in order to smooth the result, the details are retained:
+
+![ce_effect_radius_5](https://github.com/user-attachments/assets/908744b2-7bb2-4a2f-abe0-9bad8cf14740)
+
+To prevent the “bleeding” at the edge of the pumpkin caused by large radius of the guided filter, we can also increase the saturation threshold to limit the manipulation to the pumpkin, which stands out from the surroundings through saturation:
+
+![ce_effect_radius_6](https://github.com/user-attachments/assets/4806a130-7690-4204-997e-92b8232727c5)
+
+We can also use the visualization indicator to better see which areas have been changed. The pixel areas whose value has been increased are displayed in red and the areas with a reduced value are displayed in blue:
+
+![ce_effect_radius_7](https://github.com/user-attachments/assets/3fe755d4-1fee-4876-a743-bf0db4c5d319)
+
```

Comment:
As with the saturation threshold, the visualization can only be seen in areas where the blend mask selects the module's output.